content/extending-ui/extending-agent-app/index.mdx

```diff
@@ -85,4 +85,4 @@ Some applications require settings configuration, especially when they use exter
 
 We provide a special place called Settings page, where you can embed a page with configuration. It's recommended to use it together with [Sign in with LiveChat](/getting-started/authorization/sign-in-with-livechat/).
 
-See how to set up a Settings page for your app in the [documentation](/extending-ui/extending-agent-app/agent-app-sdk/#getting-started).
```

Comment:
I'm not sure why this link was changed 🤔

Reply:
Good catch, I made a mistake and replaced another link, not the one I intended. Fixed in e613edb